### PR TITLE
LOTD compatibility patches masterlist update

### PR DESCRIPTION
**Pull request by ghost**
There are a few LOTD compatibility patches I have yet to look at.  Either I couldn't find their file, it has been removed from the nexus, or it doesn't actually depend on another esp other than LOTD's own.  These include:
-DBM_Apophysis_Patch.esp
-DBM_ArtifactsOfBoethiah_Patch.esp
-DBM_NexusSkyrimOverhaul_Magic_Patch.esp
-DBM_NexusSkyrimOverhaul_Patch.esp
-DBM_StagPrinceBow_Patch.esp
-DBM_RuinsEdge_Patch.esp

The DBM_Bruma_Patch.eso and DBM_DGE_Crossbow_Patch.esp required two masters for the patch to be activated and I assumed the formatting, please be sure those are correct.

I also noticed on two of the previously listed patches there might've been a bashed tag typo with adding an extra hyphen on the Factions tag, I had corrected that at some point but please be sure this is also a correct change.

I reordered the Open Cities patch to be in alphabetical order as that is the ordering convention the LOOT seems to use for this.

Many of these patches have bashed tags already listed from the LOTD team, however I've noticed the LOOT team review files by hand and frequently add more tags.  I did not feel comfortable attempting this without first being sure of the teams bashed tag/xEdit cleaning regimen.

I can continue to work on this before it's committed but I had wanted a chance to get feedback before continuing any further.

Part of wanting to work on this is NMM/Vortex do not always behave correctly with FOMOD options optimized for MO2, and vice-versa.  This was mentioned in the LOTD discord by SirJesto and Jcbq on 7/4/18.   As such, users might not have compatibility patches auto selected and a LOOT notification of them existing would be a useful second check.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -4025,7 +4216,7 @@ plugins:
         name: 'Legacy of the Dragonborn Patches SE on Nexus Mods'
     tag:
       - Actors.AIPackages
-      - -Factions
```
Comment (MacSplody) [inline masterlist.yaml]: The extra hyphen is to signify a bad tag which is suggested to be removed.

Comment (ghost) [reply masterlist.yaml]: How do you verify a bad tag?  I noticed the xEdit script still suggested a Factions tag.

**Comment by MacSplody** [comment]
[Apophysis Dragon Priest Masks SE](https://www.nexusmods.com/skyrimspecialedition/mods/5800/) 
It's just textures so no need to worry about that one. Has an empty esp version, which just loads the BSA archive.

[Artifacts - The Tournament of the ten Bloods](https://www.nexusmods.com/skyrimspecialedition/mods/15264) 

Nexus Skyrim Overhaul & Magic version, No longer available so not much point adding them really.

[Daedric Realms - Volume I The Hunting Grounds](https://www.nexusmods.com/skyrimspecialedition/mods/3140) That's the stag bow one.

[Ruin's Edge](https://www.nexusmods.com/skyrimspecialedition/mods/12792)

**Comment by MacSplody** [comment]
So there is an error somewhere.
ERROR: boost::too_many_args: format-string referred to fewer arguments than were passed

File: masterlist.yaml [hunk 2]
```diff
@@ -3986,26 +3986,232 @@ plugins:
...
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("TheChoiceIsYours.esp") and not active("BCS_TCIY_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[The Staff of Sheogorath](https://www.nexusmods.com/skyrimspecialedition/mods/14468/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("StaffOfSheogorath.esp") and not active("DBM_StaffofSheogorath_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Ultimate Deadly Encounters - The Way of the Dovahkiin aka Sands of Time Legendary Edition ( increased spawns )](https://www.nexusmods.com/skyrimspecialedition/mods/3093/)'
```
Comment (MacSplody) [inline masterlist.yaml]: This can be shortened to Ultimate Deadly Encounters.

File: masterlist.yaml [hunk 3]
```diff
@@ -3986,26 +3986,232 @@ plugins:
...
+        condition: 'active("Dawngaurd&VolkiharArtifactsQuests.esp") and not active("DBM_DGE_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Dawnguard and Clan Volkihar Epilogues](https://www.nexusmods.com/skyrimspecialedition/mods/12098/)'
+          - '[Legendary Skyrim Crossbows SSE](https://www.nexusmods.com/skyrimspecialedition/mods/8273/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("Dawngaurd&VolkiharArtifactsQuests.esp.esp") and active("Legendary Skyrim Crossbows.esp") and not active("DBM_NAME_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Dynamic Dungeon Loot (DDL) - Random Loot Search](https://www.nexusmods.com/skyrimspecialedition/mods/10308/)'
```
Comment (MacSplody) [inline masterlist.yaml]: I'm not sure what effect `(DDL)` in the middle may have. `Dynamic Dungeon Loot` is fine without the .` (DDL) - Random Loot Search`

File: masterlist.yaml [hunk 4]
```diff
@@ -3986,26 +3986,232 @@ plugins:
...
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Beyond Reach](https://www.nexusmods.com/skyrimspecialedition/mods/3008/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("arnima.esm") and not active("DBM_BeyondReach_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Beyond Skyrim - Bruma SE](https://www.nexusmods.com/skyrimspecialedition/mods/10917/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("BSAssets.esm") and active("BSHeartland.esm") and not active("DBM_Bruma_Patch.esp")'
```
Comment (MacSplody) [inline masterlist.yaml]: You can remove `BSAssets.esm` as it's a master of `BSHeartland.esm`

File: masterlist.yaml [hunk 5]
```diff
@@ -3986,26 +3986,232 @@ plugins:
...
+        subs:
+          - '[Dawnguard and Clan Volkihar Epilogues](https://www.nexusmods.com/skyrimspecialedition/mods/12098/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("Dawngaurd&VolkiharArtifactsQuests.esp") and not active("DBM_DGE_Patch.esp")'
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Dawnguard and Clan Volkihar Epilogues](https://www.nexusmods.com/skyrimspecialedition/mods/12098/)'
+          - '[Legendary Skyrim Crossbows SSE](https://www.nexusmods.com/skyrimspecialedition/mods/8273/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
+        condition: 'active("Dawngaurd&VolkiharArtifactsQuests.esp.esp") and active("Legendary Skyrim Crossbows.esp") and not active("DBM_NAME_Patch.esp")'
```
Comment (MacSplody) [inline masterlist.yaml]: Yeah that won't work, Too many subs. It expects only two subs.

**Comment by MacSplody** [comment]
> Many of these patches have bashed tags already listed from the LOTD team, however I've noticed the LOOT team review files by hand and frequently add more tags. I did not feel comfortable attempting this without first being sure of the teams bashed tag/xEdit cleaning regimen.

[Bash Tags and the Wrye Bash Patch](http://wiki.tesnexus.com/index.php/Bash_Tags_and_the_Wrye_Bash_Patch)
Worth reading if you want an idea of when to add tags.
[Wrye Bash Advanced Readme](http://oblivionworks.sourceforge.net/readme/Wrye%20Bash%20Advanced%20Readme.html#patch-tags) as a reference.
Currently Wrye Bash for SSE only uses 
```
Delev
Invent 
Relev
NoMerge
and maybe Names & Stats.
```

Oldrim uses a lot more.
```
C.Acoustic
C.Climate
C.Encounter
C.ImageSpace
C.Light
C.Location
C.Music
C.Name
C.Owner
C.RecordFlags
C.Regions
C.SkyLighting
C.Water
Deactivate
Delev
Filter
Graphics
Invent
Names
NoMerge
Relev
Sound
```
General steps I follow are:
- Install mod.
- Run loot & sort. The unique CRC used to identify a plug-in will then be displayed. (make note of it)
- Launch xEdit, select none & then tick the plug-in you want to check.
- When it's load, run standard cleaning steps. Make note of any ITM, UDR or NAV.
- You can also check for errors & circular lvl lists, bash tags, wild edits.

It's worth reading the [LOOT documentation](https://loot.github.io/docs/)

File: masterlist.yaml [hunk 6]
```diff
@@ -3986,26 +3986,231 @@ plugins:
   - name: 'LegacyoftheDragonborn.esm'
     url: ['https://www.nexusmods.com/skyrimspecialedition/mods/11802/']
     msg:
+      - <<: *hasPatchIncluded
+        subs:
+          - '[Advanced Adversary Encounters - Ultimate SSE](https://www.nexusmods.com/skyrimspecialedition/mods/6843/)'
+          - '[Legacy of the Dragonborn SSE](https://www.nexusmods.com/skyrimspecialedition/mods/11802)'
```
Comment (MacSplody) [inline masterlist.yaml]: Just one more suggestion, `Legacy of the Dragonborn Patches` would be more appropriate for these messages.

> A patch is included with the %2% installer, please make sure you have installed the %1% patch and have it activated

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@bpipher0225 Thanks.

**Pull request merged**